gno.land/pkg/sdk/vm/keeper.go

```diff
@@ -1147,13 +1147,12 @@ func (vm *VMKeeper) processStorageDeposit(ctx sdk.Context, caller crypto.Address
 			depositAmt -= requiredDeposit
 			// Emit event for storage deposit lock
 			d := std.Coins{std.Coin{Denom: ugnot.Denom, Amount: requiredDeposit}}
-			evt := gnostd.GnoEvent{
+			evt := abci.EventKeyValue{
```

Comment:
Why this change?

I think this makes clients backwards incompatible.

Reply:
@thehowl , an alternative is for [`ExecSignAndBroadcast`](https://github.com/gnolang/gno/blob/f461665943d6c7a94efa17a2bdf648c85e9354fd/tm2/pkg/crypto/keys/client/maketx.go#L186) in tm2 to return the `DeliverTx` so that [gnokey](https://github.com/gnolang/gno/blob/f461665943d6c7a94efa17a2bdf648c85e9354fd/gno.land/pkg/keyscli/call.go#L149) can search the Events for the Gno-specific storage info. What do you think?

Reply:
Moved to [`StorageDepositEvent` and `StorageUnlockEvent`](https://github.com/gnolang/gno/blob/5d7bbbddb7b42bb5ec0e562d0ea3c5d43f6882cb/gnovm/stdlibs/std/emit_event.go#L65-L81) in gnovm `std`. See the PR description for how these are used.